tests/test_bookmark.py

```diff
@@ -84,6 +84,7 @@ def streams_to_selected_fields():
     #   and not to the last record.  Find out if this is correct and why?
     def test_first_sync_bookmark(self):
         today_datetime = dt.utcnow().replace(hour=0, minute=0, second=0, microsecond=0)
```

Comment:
Do we know the history on this?  I'm curious why we use utcnow() in the first place if we then need to replace the tz info to make it tz aware.  Is it possible we can use dt.now() instead and have the dt object be tz aware from the start?  If the answer is that we cannot use now() I would still hope that we can replace the tz info along with all the other info on the same line instead of having two lines to replace different values of the same dt object.

Reply:
Not sure why utcnow is used. This test was failing because of a change I made to the tap-tester parse_date method in the base_case file, to include tz info in the date for tap facebook table_reset_test. After discussion with Harrison decided to follow the same approach across all taps i.e to include time zone info.
Will implement it in one single line.

Reply:
Checked with dt.now() and its working fine , but still needs the tzinfo to be included.